src/app-bar.jsx

```diff
@@ -105,16 +105,22 @@ const AppBar = React.createClass({
 
     /**
      * Callback function for when the left icon is selected via a touch tap.
+     *
+     * @param {object} event TouchTap event targeting the left `IconButton`.
```

Comment:
Yeah, that's no problem; I was just going by the convention being used [here](http://www.material-ui.com/#/components/snackbar).